rsearch/Godeps/Godeps.json

```diff
@@ -0,0 +1,23 @@
+{
+	"ImportPath": "github.com/romana/contrib/rsearch",
```

Comment:
This should probably be the target ImportPath (ie: `k8s.io/contrib/rsearch`) before it's submitted upstream.